File: ApnaCollege/4Loop/ReverseNumber.java
import java.util.*;
public class ReverseNumber {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    System.out.println("Enter the number:- ");
    int num1 = sc.nextInt();
    int reverse = 0;
    while (num1 != 0) {
      int lastdigit = num1 % 10;
      reverse = reverse * 10 + lastdigit;
      num1 = num1 / 10;
    }
    System.out.println("Reverse of a number is:- " + reverse);
    sc.close();
  }
}

File: basic/reverseNumber.cpp
/* Statement:- Program to reverse a number */
/* ---------------------------------------- */
/* Created By Visual Studio Code
Author:- Faraz Alam
Email:- farazalam2017@gmail.com
Date: 2025-02-04 09:33:01
Title: reverseNumber.cpp */
/* ---------------------------------------- */
/* ---------------------------------------- */
#include <iostream>
using namespace std;
int main()
{
  int number;
  int reverseNumber = 0;
  cout << "Enter the number:- ";
  cin >> number;
  int originalNumber = number;
  while (number > 0)
  {
    int lastDigit = number % 10;
    reverseNumber = (reverseNumber * 10) + lastDigit;
    number = number / 10;
  }
  cout << "Reverse of " << originalNumber << " is " << reverseNumber;
}